test/concerns/validations_test.rb

```diff
@@ -8,42 +8,29 @@ def test_ancestry_column_validation
         node.send :write_attribute, model.ancestry_column, value
         node.valid?; assert node.errors[model.ancestry_column].blank?
       end
-      ['1/3/', '/2/3', 'a', 'a/b', '-34', '/54'].each do |value|
```

Comment:
we used to focus that the format of this field was serialized correctly.

The new functionality parses these values just fine.
And since we are moving the serialization to a single method - the chances of failure is much lower than when this field was formed all over the code.